core/src/main/scala/org/apache/spark/network/Security.scala

```diff
@@ -35,33 +37,15 @@ object Security extends Logging {
     }
   }
 
-  def enableSSL(spark: SparkSession, conf: H2OConf): Unit = enableSSL(spark, conf.sparkConf)
+  def enableSSL(spark: SparkSession, conf: H2OConf): Unit = enableSSL(conf)
 
-  def enableSSL(spark: SparkSession): Unit = enableSSL(spark, spark.sparkContext.conf)
-
-  def enableFlowSSL(spark: SparkSession, conf: H2OConf): H2OConf = {
+  def enableFlowSSL(conf: H2OConf): H2OConf = {
     val sslPair = SparklingWaterSecurityUtils.generateSSLPair(namePrefix = "h2o-internal-auto-flow-ssl")
     conf.setJks(sslPair.jks.getLocation)
     conf.setJksPass(sslPair.jks.pass)
   }
 
-  def enableSSL(sc: SparkContext, conf: SparkConf): Unit = {
```

Comment:
Removing as methods accepting only spark conf did not have any affect, we need to update H2OConf at this stage